### ISSUE TITLE
Dockerfile generation

### ISSUE THREAD

**Issue by szaibot** [orctom/gradle-archetype-plugin#1]
Hi,
I have a problem with Dockerfile generation. Dockerfile has no extension by convention, but this plugin treats files without extensions like directories. Is there any chance to fix it?

The problem is in this line of code (FileUtils:101):

```
static boolean isDir(File file) {
     !file.name.contains('.')
}
```

I could make some workaround/fix but I'm not a groovy master ;)

**Comment by orctom** [comment]
Sorry replying you so late.
I just released version 1.3, it should be fixed. Check it out:)

**Comment by mikexliu** [comment]
@orctom 
Thanks for the fix here, works for me now too.

It seems like `gradlew` doesn't work though. I'm using a work-around with
```
task create(type: Copy, dependsOn: "generate") {
    from(".")
    into("${System.properties['target']}/${System.properties['name']}")
    include("gradlew")
}
```

Let me know if this should be a separate issue. I didn't get a chance to look into the code to submit a pull request yet, sorry!

**Comment by orctom** [comment]
You could tell the plugin that `gradlew` is not a template file, by ```echo gradlew >> .nontemplate```.

**Comment by mikexliu** [comment]
Ahh I thought I tried that already and hit an issue with permissions before.

Thanks!!

**Comment by orctom** [comment]
It should be fixed in version `1.3`, please give it a shot. The plugin is just copying the `nontemplates` to the `generated` directory, so the permission is kept.
The bug been fixed in 1.3 is that the way how it determines a given path as a file or directory, it was treating files with no extensions as directories.
